soapy/confParse.py

```diff
@@ -788,6 +798,11 @@ class WfsConfig(ConfigObj):
                         ("nx_guard_pixels", 0),
                         ("loadModule", None),
                         ("photometric_zp", 2e9)
```

Comment:
You forgot to add a comma here! I have given it a quick test on my machine and if you add that then all the tests should pass.